flutter_local_notifications/README.md

```diff
@@ -59,8 +59,9 @@ A cross platform plugin for displaying local notifications.
 * **iOS**. Uses the [UserNotification APIs](https://developer.apple.com/documentation/usernotifications) (aka the User Notifications Framework)
 * **macOS**. On macOS versions older than 10.14, the plugin will use the [NSUserNotification APIs](https://developer.apple.com/documentation/foundation/nsusernotification). The [UserNotification APIs](https://developer.apple.com/documentation/usernotifications) (aka the User Notifications Framework) is used on macOS 10.14 or newer. Notification actions only work on macOS 10.14 or newer
 * **Linux**. Uses the [Desktop Notifications Specification](https://specifications.freedesktop.org/notification-spec/)
+* **Windows** Uses the [C++/WinRT](https://learn.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/) implementation of [Toast Notifications](*https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/toast-notifications-overview)
```

Comment:
There's an asterisk before the link https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/toast-notifications-overview. I believe is causing the link to not appear properly as shown on your fork's readme https://github.com/Levi-Lesches/flutter_local_notifications/tree/windows-ffi/flutter_local_notifications

Reply:
Good catch! Not on my work laptop right now but I'll fix it tomorrow.